kyo-reactive-streams/shared/src/main/scala/kyo/interop/flow/package.scala

```diff
@@ -0,0 +1,33 @@
+package kyo.interop
+
+import java.util.concurrent.Flow.*
+import kyo.*
+import kyo.interop.reactivestreams
+import kyo.kernel.Boundary
+import org.reactivestreams.FlowAdapters
+
+package object flow:
+    inline def fromPublisher[T](
+        publisher: Publisher[T],
+        bufferSize: Int
+    )(
+        using
+        Frame,
+        Tag[T]
+    ): Stream[T, Async] < IO = reactivestreams.fromPublisher(FlowAdapters.toPublisher(publisher), bufferSize)
+
+    def subscribeToStream[T, Ctx](
+        stream: Stream[T, Ctx],
+        subscriber: Subscriber[? >: T]
+    )(
+        using
+        Boundary[Ctx, IO],
```

Comment:
We have some inference issues when using `Boundary` directly. We've been using a workaround with a user-facing `inline` method calling another private one that then requires the `Boundary` evidence. See `Async` for examples.